pandas/core/window.py

```diff
@@ -1354,6 +1354,8 @@ def _get_center_of_mass(com, span, halflife, alpha):
 
 
 def _offset(window, center):
+    if com.is_float(window):
+        raise TypeError("Window should be of type Integer, given Float")
```

Comment:
It would be better to use `if not com.is_integer(window): raise TypeError` but that will clearly need to happen at a another point in the code, because at this point we accept arrays (see line below)